keps/sig-api-machinery/5000-api-linting-crd-schema-tooling/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: API Linting and CRD Schema Change Tooling
+kep-number: 5000
+authors:
+  - "@everettraven"
+  - "@JoelSpeed"
+owning-sig: sig-api-machinery
+participating-sigs:
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-01-29
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
you need sponsor from the sig leads
/assign @deads2k @jpbetz

Reply:
Thanks!

Went ahead and updated this to include them both in the reviewers and approvers list.